shablona/__init__.py

```diff
@@ -1,2 +1,2 @@
-from .shablona import *  # noqa
 from .version import __version__  # noqa
+from .shablona import *  # noqa
```

Comment:
BTW having import of version first is important to have access to `__version__` while importing submodules